pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -280,111 +114,62 @@ var (
-		TLSBootstrappingEnabled:         true,                                                                // s
-		SecureTLSBootstrappingEnabled:   false,                                                               // s
-		THPEnabled:                      "",                                                                  // cd
-		THPDefrag:                       "",                                                                  // cd
-		ServicePrincipalFileContent:     base64.StdEncoding.EncodeToString([]byte("msi")),                    // s
-		KubeletClientContent:            "",                                                                  // -
-		KubeletClientCertContent:        "",                                                                  // -
-		KubeletConfigFileEnabled:        false,                                                               // s
-		KubeletConfigFileContent:        "",                                                                  // s
-		SwapFileSizeMB:                  0,                                                                   // td
-		GPUInstanceProfile:              "",                                                                  // td
-		CustomSearchDomainName:          "",                                                                  // cd
-		CustomSearchRealmUser:           "",                                                                  // cd
-		CustomSearchRealmPassword:       "",                                                                  // cd
-		MessageOfTheDay:                 "",                                                                  // td
-		HasKubeletDiskType:              false,                                                               // td
-		SysctlContent:                   base64.StdEncoding.EncodeToString(sysctlContent),                    // td
-		KubeletFlags:                    "",                                                                  // psX
-		AzureEnvironmentFilepath:        "",                                                                  // s
-		KubenetTemplate:                 base64.StdEncoding.EncodeToString(kubenetTemplate),                  // s
-		ContainerdConfigContent:         "",                                                                  // kd
-		IsKata:                          false,                                                               // n
-
+	staticNodeBootstrapVars = nbcontractv1.Configuration{
+		ClusterConfig: &nbcontractv1.ClusterConfig{
+			VmType:              nbcontractv1.ClusterConfig_VMSS, // xd
+			UseInstanceMetadata: true,                            // s
+			LoadBalancerConfig: &nbcontractv1.LoadBalancerConfig{
+				LoadBalancerSku:                       nbcontractv1.GetLoadBalancerSKU("Standard"), // xd
+				ExcludeMasterFromStandardLoadBalancer: to.BoolPtr(true),                            //s
```

Comment:
same nits please us lo rather than to

Reply:
why do we prefer lo rather than to?

Reply:
just wider usage I would like to remove to eventually